### PR TITLE
Landing page and fabric script to build redux apps

### PR DESCRIPTION
**Pull request by rlr**
Fixes issue #173 and issue #174.

To try it out (npm and gulp are required):

    $ fab build_ui
    $ manage.py runserver
    $ open http://localhost:5000/

For the new splice ui (campaign manager), there is a problem with background images. They are specified in SASS as absolute paths like `/public/img/filename.jpg` and we are serving them from `/static/build/....`. That's an outstanding issue I'll need to figure out with @tkiethanom. But this gets us closer.

r? @ncloudioj @tspurway

### CODE REVIEW

**Comment by rlr** [comment]
NOTE: all the code in ui/ had been previously reviewed in the master branch.

File: fabfile.py [hunk 1]
```diff
@@ -92,6 +95,18 @@ def deploy_cold():
     set_symlinks()
 
 
+def build_ui():
+    with lcd('ui'):
+        local('pwd')
+        local('npm install')
+        local('npm run-script bundle')
```
Comment (k88hudson) [inline fabfile.py]: you can use `run` instead of `run-script`

File: fabfile.py [hunk 2]
```diff
@@ -92,6 +95,18 @@ def deploy_cold():
...
+
+    with lcd('front_end'):
+        local('npm install')
+        local('gulp build:dist')
```
Comment (k88hudson) [inline fabfile.py]: This requires a global install of `gulp`. Wouldn't it be better if we just installed it locally and called this command in the script?

File: fabfile.py (hunk 1 above)
Comment (rlr) [inline fabfile.py]: oops. just realized i left in a PWD

File: fabfile.py (hunk 2 above)
Comment (rlr) [inline fabfile.py]: That's how TBG has set up the new app. We can change it later. I'd like to remove the gulp dependency.

Comment (k88hudson) [inline fabfile.py]: I don't mean switch from gulp completely, just this: https://github.com/oyiptong/splice/pull/215/files

**Comment by rlr** [comment]
Fixed those comments ^^. They depend on landing pull #215.

**Comment by rlr** [comment]
I've rebased again. Should be good for another r? pass now.

**Comment by ncloudioj** [comment]
It looks good to me. r+!

We can drop some "# pragma: no cover" in the `view.py` to make the coveralls happy.

**Pull request closed without merging**